subsys/net/lib/aws_iot/src/aws_iot.c

```diff
@@ -103,6 +106,11 @@ static char delete_rejected_topic[DELETE_REJECTED_TOPIC_LEN + 1];
 /* Empty string used to request the AWS IoT shadow document. */
 #define AWS_IOT_SHADOW_REQUEST_STRING ""
 
+#define AWS_BROKER_HOST_NAME_PREFIX "%s"
```

Comment:
To be honest I find this define name a bit confusting, it's not really a prefix, nothing is added to the hostname.